content/en/docs/ops/best-practices/image-signing-validation/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Image Signing And Validation
+description: Describes how to use image signatures to verify the provenance of Istio images.
+weight: 35
+aliases:
+    - /docs/ops/best-practices/image-signing-validation/
+keywords: [install]
+owner: istio/wg-environments-maintainers
```

Comment:
Should environments own this page?

Reply:
I wasn't sure, here. AFAIK no existing pages are owned by test and release, and it wasn't clear if this was a group that was internal to the istio.io repo or if it was referencing github org groups.

Reply:
It would refer to the WG that would approve changes to this page (or be responsible if a test suddenly started failing (noting no test on this page so can ignore that)). A doc reviewer, may not understand the technicalities of the page so would expect that someone from the specified WG would approve the PR before it would be approved by the doc reviewer and merge.

Reply:
The environments group will take ownership of this page. That's fine with me.

Reply:
Leaving as Environments, then.